Alexey-budgetCalc/modul.py

```diff
@@ -0,0 +1,211 @@
+import csv
+import matplotlib.pyplot as plt
+
+
+def month_input():
+    month = ""
+    flag = True
+    while flag:
+        month = input("Enter the month in format YYYY-MM \n")
+        if (len(month) == 7) and (month[4] == "-"):
+            if check_int(month[5:6]) and check_int(month[:3]):
+                flag = False
+    return month
+
+
+def date_input():
+    date = ""
+    flag = True
+    while flag:
+        date = input("Enter the date in format YYYY-MM-DD \n")
+        if (len(date) == 10) and (date[4] == "-") and (date[7] == "-"):
+            if check_int(date[8:9]) and check_int(date[5:6]) and check_int(date[:3]):
+                flag = False
+    return date
+
+
+def int_pos_input():
+    number = 0
+    while number < 1:
+        number = input("Input positive mumber\n")
+        try:
+            number = int(number)
+        except Exception:
+            print("Try again\n")
+            number = 0
+    return number
+
+
+def int_input():
+    flag = True
+    number: int = 0
+    while flag:
+        string = input("Input positive mumber\n")
+        try:
+            number = int(string)
+            flag = False
+        except Exception:
+            print("Try again\n")
+            flag = True
+    return number
+
+
+def check_int(number):
+    flag = True
+    try:
+        int(number)
+    except Exception:
+        flag = False
+    return flag
+
+
+def show_simple_sort(info_dict):
+    labels = list(info_dict.keys())
+    sizes = list(info_dict.values())
+    fig1, ax1 = plt.subplots()
+    ax1.pie(sizes, labels=labels, autopct="%1.1f%%", shadow=True, startangle=90)
+    ax1.axis("equal")
+    plt.show()
+
+
+def show_bigger_sort(info_dict):
+    labels = []
+    sizes = []
+    for category in info_dict.keys():
+        prelable = list(info_dict[category].keys())
+        for index in range(len(prelable)):
```

Comment:
More pytonic way is to use `for idx, element in enumerate(some_list))` to get indexes and values, but not range